docs/cli.md

```diff
@@ -8,52 +8,65 @@ last_update:
+This document provides guidance on managing metadata within Apache Gravitino using the Command Line Interface (CLI). The CLI offers a terminal based alternative to using code or the REST interface for metadata management.
 
-Currently, you can view basic metadata information for metalakes, catalogs, schema, and tables. The ability to create, update, and delete metalakes and support additional entities in planned in the near future.
+Currently, the CLI allows users to view metadata information for metalakes, catalogs, schemas, and tables. Future updates will expand on these capabilities to include roles, users, and tags.
 
 ## Running the CLI
 
-You can set up an alias for the command like so:
+You can configure an alias for the CLI for ease of use, with the following command:
 
 ```bash
 alias gcli='java -jar ../../cli/build/libs/gravitino-cli-*-incubating-SNAPSHOT.jar'
 ```
 
-Or use the `gcli.sh` script found in the `clients/cli/bin/` directory to run the CLI.
+Or you use the `gcli.sh` script found in the `clients/cli/bin/` directory to run the CLI.
 
 ## Usage
 
- To run the Gravitino CLI, use the following command structure:
+The general structure for running commands with the Gravitino CLI is `gcli entity command [options]`.
 
  ```bash
- usage: gcli [metalake|catalog|schema|table] [list|details|create|delete|update] [options]
+ usage: gcli [metalake|catalog|schema|table|column] [list|details|create|delete|update|set|remove|properties] [options]
  Options
- -f,--name <arg>       full entity name (dot separated)
- -h,--help             command help information
- -i,--ignore           Ignore client/sever version check
- -m,--metalake <arg>   Metalake name
- -r,--server           Gravitino server version
- -u,--url <arg>        Gravitino URL (default: http://localhost:8090)
- -v,--version          Gravitino client version
+ -b,--bootstrap <arg>   Kafka bootstrap servers
+ -c,--comment <arg>     entity comment
+ -d,--database <arg>    database name
+ -h,--help              command help information
+ -i,--ignore            Ignore client/sever version check
+ -j,--jdbcurl <arg>     JDBC URL
+ -l,--user <arg>        database username
+ -m,--metastore <arg>   Hive metastore URI
+ -n,--name <arg>        full entity name (dot separated)
+ -p,--provider <arg>    provider one of hadoop, hive, mysql, postgres,
+                        iceberg, kafka
+ -r,--rename <arg>      new entity name
+ -u,--url <arg>         Gravitino URL (default: http://localhost:8090)
+ -v,--value <arg>       property value
+ -w,--warehouse <arg>   warehouse name
+ -z,--password <arg>    database password
  ```
 
 ## Commands
 
-The following commands are available for entity management:
+The following commands are used for entity management:
 
 - list: List available entities
 - details: Show detailed information about an entity
 - create: Create a new entity
 - delete: Delete an existing entity
 - update: Update an existing entity
+- set: Set an entities property
```

Comment:
"an entities property", is this grammatically correct?

Reply:
Yes in general words that start with a vowel use "an" rather than "a" and words that end in y the plural is "ies". But I can see that may be unclear so I've changed it to:
- set: Set a property on an entity
- remove: Remove a property from an entity

Reply:
OK, thanks for the confirmation; It ups to you, I'm fine with both.